kubernetes/samples/charts/apache-webtier/values.yaml

```diff
@@ -39,3 +39,39 @@ httpNodePort: 30305
 
 # NodePort to expose for https access
 httpsNodePort: 30443
+
+# The VirtualHostName of the Apache HTTP server. It is used to enable custom SSL configuration.
+# If it is set, the Apache HTTP Server is configured to listen to port 4433 for SSL traffic.
+virtualHostName:
+
+# The customer supplied certificate to use for Apache webtier SSL configuration.
+# The value must be a string containing a base64 encoded certificate.
+customCert:
```

Comment:
If the customer does not provide the SSL cert and key, does the sample use the default built-in auto-generated sample cert and key in the apache image? if so, we need to document this.

Reply:
We could also document the steps to generate the cert and key using openssl. 
In addition, I am wondering if it is easier to ask customers to provide the cert and key directly here, or to provide the paths to the cert and key files.

Reply:
Correct. If the customer enable custom SSL configuration and does not provide the cert and private key, the sample will use the default built-in auto-generated sample cert and key in the apache image.
The behavior is same as the weblogic-operator helm chart, both require that the customer provide base64 encoded string. It is not complex. Execute following command will get the base64 encoded certificate.
base64 -i ${cert_file} | tr -d '\n'